lib/pseudolocalization.rb

```diff
@@ -23,8 +26,28 @@ def respond_to_missing?(name, include_private = false)
       end
 
       def translate(locale, key, options)
+        return original_backend.translate(locale, key, options) if key_ignored?(key)
+
         ::Pseudolocalization::I18n::Pseudolocalizer.pseudolocalize(original_backend.translate(locale, key, options))
       end
+
+      private
+
+      def key_ignored?(key)
+        return false unless ignores
+
+        ignores.any? do |ignore|
+          case ignore
+          when Regexp
+            key.to_s.match(ignore)
+          when String
+            File.fnmatch(ignore, key.to_s)
```

Comment:
While this lives under file, it works with normal strings as well and therefore we get globing for free